### PR TITLE
feat(metric-explorer-l-s-b): renewal metric lsb (inventory v2)

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Self-reviewed (coding conventions, bug-free, functionality verified, tests checked, documentation updated)
- [ ] Minor change, review optional (`style`, `chore`, `ci`, straightforward changes, etc.)
- [ ] Previously reviewed in feature branch, no further review needed
- [ ] Need review or discussion

### Description (optional)
- Apply new namespaceGroup/namespace/metric (inventory v2) to MetricExplorerLSB.
- Replace `reference-store(namespace, metric, metric-example)` to realtime fetching.
- Apply `namespaceGroup`

image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/components/MetricExplorerLSBMetric.vue [hunk 1]
```diff
@@ -64,27 +67,27 @@ const state = reactive({
...
         return sortedMetrics.map((metric) => {
             const metricTreeNode = {
-                id: metric.key,
+                id: metric.metric_id,
                 depth: 0,
                 data: {
                     ...metric,
                     type: 'metric',
-                    is_managed: metric.data.is_managed,
+                    is_managed: metric.is_managed,
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/components/MetricExplorerLSBMetric.vue:81]: nitpick: it looks unnecessary 😮

File: apps/web/src/services/asset-inventory/components/MetricExplorerLSBMetric.vue [hunk 2]
```diff
@@ -148,10 +150,11 @@ watch(() => route.params, () => {
...
 /* Watcher */
-watch(() => storeState.selectedNamespace, async (selectedNamespace) => {
-    if (!isEmpty(selectedNamespace)) {
-        await allReferenceStore.load('metric', { force: true });
-        await metricExplorerPageStore.loadMetricExamples(selectedNamespace?.name);
+watch(() => metricExplorerLSBState.selectedNamespaceId, async (selectedNamespaceId) => {
+    if (selectedNamespaceId) {
+        await metricExplorerLSBStore.loadNamespace(selectedNamespaceId);
+        await metricExplorerLSBStore.loadMetricList(selectedNamespaceId);
+        await metricExplorerLSBStore.loadMetricExampleList(selectedNamespaceId);
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/components/MetricExplorerLSBMetric.vue:157]: nitpick: I think those can be executed parallel 😊

**Review by WANZARGEN — APPROVED** [review]
😊

**Review by sulmoJ — APPROVED** [review]
LGTM~

**Pull request merged**